test/WebJobs.Script.Tests/Diagnostics/LinuxAppServiceEventGeneratorTests.cs

```diff
@@ -175,31 +175,31 @@ public void ParseAzureMonitoringEvents(LogLevel level, string resourceId, string
                 p => Assert.True(DateTime.TryParse(p, out DateTime dt)));
         }
 
-        [Theory]
-        [MemberData(nameof(LinuxEventGeneratorTestData.GetFunctionExecutionEvents), MemberType = typeof(LinuxEventGeneratorTestData))]
-        public void ParseFunctionExecutionEvents(string executionId, string siteName, int concurrency, string functionName, string invocationId,
-            string executionStage, long executionTimeSpan, bool success)
-        {
-            _generator.LogFunctionExecutionEvent(executionId, siteName, concurrency, functionName, invocationId, executionStage, executionTimeSpan, success);
-            string evt = _loggers[LinuxEventGenerator.FunctionsExecutionEventsCategory].Events.Single();
-
-            Regex regex = new Regex(LinuxAppServiceEventGenerator.ExecutionEventRegex);
-            var match = regex.Match(evt);
-
-            Assert.True(match.Success);
-            Assert.Equal(10, match.Groups.Count);
-
-            var groupMatches = match.Groups.Cast<Group>().Select(p => p.Value).Skip(1).ToArray();
-            Assert.Collection(groupMatches,
-                p => Assert.Equal(executionId, p),
-                p => Assert.Equal(siteName, p),
-                p => Assert.Equal(concurrency.ToString(), p),
-                p => Assert.Equal(functionName, p),
-                p => Assert.Equal(invocationId, p),
-                p => Assert.Equal(executionStage, p),
-                p => Assert.Equal(executionTimeSpan.ToString(), p),
-                p => Assert.True(Convert.ToBoolean(p)),
-                p => Assert.True(DateTime.TryParse(p, out DateTime dt)));
-        }
+        // [Theory]
+        // [MemberData(nameof(LinuxEventGeneratorTestData.GetFunctionExecutionEvents), MemberType = typeof(LinuxEventGeneratorTestData))]
+        // public void ParseFunctionExecutionEvents(string executionId, string siteName, int concurrency, string functionName, string invocationId,
+        //     string executionStage, long executionTimeSpan, bool success)
+        // {
```

Comment:
You can test by setting the feature flag and seeing what the outcome is (with and without the flag)

Reply:
Example of setting the feature flag in a test

https://github.com/Azure/azure-functions-host/blob/9a776f22ed6b85fc026a2799f3b0d4b8dd87a9ad/test/WebJobs.Script.Tests/UtilityTests.cs#L866

Reply:
Indeed. The test should be updated, but similar to the comment in the other PR, we should not be checking commented out code in.

Reply:
Updated the tests in line with other PR